### PR TITLE
ECS EC2 Cluster - traces

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

https://jira.kumoroku.com/jira/browse/SUMO-211049

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by pbrzoska-sumo** [comment]
Only formatting changes or anything changed in the content ?

**Review by mat-rumian — APPROVED** [review]

File: docs/apm/traces/get-started-transaction-tracing/set-up-traces-collection-aws-environments.md [hunk 1]
```diff
@@ -30,155 +30,107 @@ instances ([Scenario 1 below](#scenario-1-aws-opentelemetry-collector-installati
...
+1. The next step is to check if your deployment is properly running. Go to the [ECS Console](https://console.aws.amazon.com/ecs/home), select the proper region, and select the cluster you used to deploy the AWS OpenTelemetry Collector. Navigate to the **Tasks** tab and check if the task is running. <br/>  ![task status running.png](/img/traces/task-status-running.png)
+1. Finally, click on the task and expand the **Containers** list. In the **Network Bindings > External Link** section, you'll find the information on where to send telemetry data.<br/>![external links.png](/img/traces/external-links.png)
+  If you plan to send data to the ECS Collector service from a container running in the bridge network mode using the same host, you can use Docker Gateway IP - 172.17.0.1 on EC2 Linux in the application environment variables. For example:
+  ```bash
+  OTEL_EXPORTER_OTLP_ENDPOINT=http://172.17.0.1:55681
+  OTEL_TRACES_EXPORTER=otlp
+  ```
+  However, with this configuration, the collector service task must be running in each of the used EC2 instances.
```
Comment (kimsauce) [inline docs/apm/traces/get-started-transaction-tracing/set-up-traces-collection-aws-environments.md:69]: @pbrzoska-sumo To clarify, the actual change to this PR was this block of text.

**Comment by kimsauce** [comment]
> Only formatting changes or anything changed in the content ?

Sorry I forgot to clarify - the actual change to this PR was this block of text (lines 64-69).

File: docs/apm/traces/get-started-transaction-tracing/set-up-traces-collection-aws-environments.md (hunk 1 above)
Comment (pbrzoska-sumo) [reply docs/apm/traces/get-started-transaction-tracing/set-up-traces-collection-aws-environments.md:69]: ok, if that comes from @mat-rumian (it was not clear from the initial PR), then that's fine as he already approved.

**Review by pbrzoska-sumo — APPROVED** [review]

**Pull request merged**